packages/plugin-ext-vscode/src/browser/plugin-vscode-commands-contribution.ts

```diff
@@ -309,6 +309,22 @@ export class PluginVscodeCommandsContribution implements CommandContribution {
          * Show Opened File in New Window	workbench.action.files.showOpenedFileInNewWindow
          * Compare Opened File With	workbench.files.action.compareFileWith
          */
+
+        commands.registerCommand({ id: 'editor.action.rename' }, {
```

Comment:
hello, are these commands should be handled more globally https://github.com/theia-ide/theia/blob/33590a9a8fc9899fdd0f0ed57cb3ea4871adf6e1/packages/monaco/src/browser/monaco-command.ts ?

Reply:
@benoitf Fixed. I wasn't exactly sure where to define the commands so I defined them with the editor commands. If they should be somewhere else let me know!